stable/ipfs/templates/service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "servicename" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  type: {{ .Values.service.type }}
```

Comment:
Does the service need to be headless? We also need a service anyway, and this paradigm works well - ipfs.namespace round robins to one of them, and you can do ipfs-0.ipfs or ipfs-1.ipfs for each one if you need.